src/Redux/books/book.js

```diff
@@ -0,0 +1,32 @@
+import { data } from '../../Components/data';
+/* eslint-disable no-case-declarations */
```

Comment:
The goal of linters is to help us write clean code. Kindly remove linters disabling here.